pkg/cmd/roachtest/cluster.go

```diff
@@ -2321,25 +2321,37 @@ func (c *clusterImpl) RunE(ctx context.Context, options install.RunOptions, args
 // you treat an error from the command. This makes error checking easier / friendlier
 // and helps us avoid code replication.
 func (c *clusterImpl) RunWithDetailsSingleNode(
-	ctx context.Context, testLogger *logger.Logger, nodes option.NodeListOption, args ...string,
+	ctx context.Context, testLogger *logger.Logger, options install.RunOptions, args ...string,
 ) (install.RunResultDetails, error) {
+
+	nodes := option.NodeListOption{}
+	for _, n := range options.Nodes {
+		nodes = append(nodes, int(n))
+	}
```

Comment:
Heh, this kind of code is so silly, we should have used a single `[]int` custom type (at most) between roachprod and roachtest from the beginning 🤦.

We might be interested in having  a "conversion" function that does this for us, just like we have `InstallNodes()` to do the conversion in the opposite direction currently. Optional.